mod_ci/controllers.py

```diff
@@ -622,19 +621,25 @@ def start_ci():
 
 
 def update_build_badge(status, test):
+    '''
+    Build status badge for current test to be displayed on sample-platform.ccextractor.org
+
+    :param status: current testing status
+    :type status: str
+    :param test: current commit that is tested
+    :type test: str
```

Comment:
This is incorrect. If `test` was a `str`, then `test.test_type` is not a valid code construct.